swift/src/IceImpl/include/CtrlCHandler.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) ZeroC, Inc.
+
+#import "Config.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// The implementation of Ice.CtrlCHandler, which wraps the C++ class Ice::CtrlCHandler.
+ICEIMPL_API @interface ICECtrlCHandler : NSObject
+- (void)receiveSignal:(void (^)(int))callback;
+@end
+
+#ifdef __cplusplus
+
+@interface ICECtrlCHandler ()
+{
+    Ice::CtrlCHandler cppObject;
```

Comment:
It's essential to use a field and not a property here, since a property wants to return a copy of the C++ object (which is not copyable).